examples/troopjs_require/readme.md

```diff
@@ -1,16 +1,21 @@
-# TroopJS TodoMVC Example
+# TroopJS • [TodoMVC](http://todomvc.com)
```

Comment:
All changes for `readme.md` are already in a separate commit. Did you mean separate PR?